lib/node_modules/@stdlib/stats/base/dists/kumaraswamy/README.md

```diff
@@ -111,10 +111,74 @@ var objectKeys = require( '@stdlib/utils/keys' );
 var kumaraswamy = require( '@stdlib/stats/base/dists/kumaraswamy' );
 
 console.log( objectKeys( kumaraswamy ) );
+// Create a Kumaraswamy distribution object
+var a = 2.0;
+var b = 5.0;
+var dist = new kumaraswamy.Kumaraswamy( a, b );
+
+// Calculate basic distribution properties
+console.log( 'Mean: %d', dist.mean );
+console.log( 'Median: %d', dist.median );
+console.log( 'Mode: %d', dist.mode );
+console.log( 'Variance: %d', dist.variance );
+
+// Evaluate the probability density function (PDF)
+var x = 0.5;
+var y = dist.pdf( x );
+console.log( 'PDF at x = %d: %d', x, y );
+
+// Evaluate the cumulative distribution function (CDF)
+y = dist.cdf( x );
+console.log( 'CDF at x = %d: %d', x, y );
+
+// Evaluate the natural logarithm of PDF and CDF
+console.log( 'Log PDF at x = %d: %d', x, dist.logpdf( x ) );
+console.log( 'Log CDF at x = %d: %d', x, dist.logcdf( x ) );
+
+// Calculate the quantile for a given probability
+var p = 0.75;
+x = dist.quantile( p );
+console.log( 'Quantile at p = %d: %d', p, x );
+
+// Generate random numbers
+var r = dist.random( 5 );
```

Comment:
I don't recall `.random` being a method on the distribution object.

Reply:
updated !